README.md

```diff
@@ -6,112 +6,79 @@
+    - random seed in task
+    - displayed log
+    - all parameters set as class variables in the task
+- If change parameter of Task, automatically rerun.
+- Support GCS or S3
+- The above output is exchanged between tasks as an intermediate file, which is memory-friendly
+- pandas.DataFrame type and column checking during I/O
+- Directory structure of saved files is automatically determined from structure of script
+- Seeds for numpy and random are automatically fixed
+- Can code while adhering to SOLID principles as much as possible
+- Tasks are locked via redis even if they run in parallel
 
-    def output(self):
-        # please use TaskOnKart.make_target to make Target.
-        return self.make_target('basic_task.csv')
+*These are all functions baptized for creating Machine Learning batches. Provides an excellent environment for reproducibility and team development.*
 
-    def run(self):
-        # load data which TaskA output
-        texts = self.load()
 
-        # do something with texts, and make results.
+# Getting Started
 
-        # save results with the file path {self.workspace_directory}/basic_task_{unique_id}.csv
-        self.dump(results)
+Within the activated Python environment, use the following command to install gokart.
+
+```
+pip install gokart
 ```
 
-### Details of base functions
-#### Make Target with TaskOnKart
-`TaskOnKart.make_target` judge `Target` type by the passed path extension. The following extensions are supported.
 
- - pkl
- - txt
- - csv
- - tsv
- - gz
- - json
- - xml
+# Quickstart
+
+A minimal gokart tasks looks something like this:
+
 
-#### Make Target for models which generate multiple files in saving.
-`TaskOnKart.make_model_target` and `TaskOnKart.dump` are designed to save and load models like gensim.model.Word2vec.
 ```python
-class TrainWord2Vec(TaskOnKart):
-    def output(self):
-        # please use 'zip'.
-        return self.make_model_target(
-            'model.zip',
-            save_function=gensim.model.Word2Vec.save,
-            load_function=gensim.model.Word2Vec.load)
+import gokart
 
+class Example(gokart.TaskOnKart):
     def run(self):
-        # make word2vec
-        self.dump(word2vec)
+        self.dump('Hello, world!')
+
+task = Example()
+output = gokart.build(task)
+print(output)
 ```
 
-#### Load input data
-##### Pattern 1: Load input data individually.
-```python
-def requires(self):
-    return dict(data=LoadItemData(), model=LoadModel())
+`gokart.build` return the result of dump by `gokart.TaskOnKart`. The example will output the following.
 
-def run(self):
-    # pass a key in the dictionary `self.requires()`
-    data = self.load('data')
-    model = self.load('model')
-```
 
-##### Pattern 2: Load input data at once
-```python
-def run(self):
-    input_data = self.load()
-    """
-    The above line is equivalent to the following:
-    input_data = dict(data=self.load('data'), model=self.load('model'))
-    """
+```
+Hello, world!
 ```
 
```

Comment:
Which document should I look for next step?
( e.g. defining task argument, make task idempotent, ... )

Also, because link to documentation folder is to far, readers could lost way.

If there are some links to specific document, it helps lot.

Reply:
Thanks. added